homeassistant/components/homeassistant/scene.py

```diff
@@ -54,6 +54,21 @@ def _convert_states(states):
     return result
 
 
+async def _part_of_config(hass, name):
+    """Determine whether entity_id was configured manually, return True if an error occurs."""
+    try:
+        conf = await conf_util.async_hass_config_yaml(hass)
```

Comment:
The YAML can be out of sync with what is loaded (in case of a reload etc). We should determine it differently. Maybe have the entity have a property? `platform.entities.get(entity_id).from_yaml`

Reply:
Thanks, that was a good hint. It makes everything much easier :+1: